tests/e2e/e2e_test.go

```diff
@@ -25,6 +25,33 @@ import (
 	"github.com/osmosis-labs/osmosis/v14/tests/e2e/initialization"
 )
 
+func (s *IntegrationTestSuite) TestConcentratedLiquidity() {
+	chainA := s.configurer.GetChainConfig(0)
+	node, err := chainA.GetDefaultNode()
+	s.Require().NoError(err)
+
+	var (
+		denom0                    string = "uosmo"
+		denom1                    string = "uion"
+		tickSpacing               uint64 = 1
+		precisionFactorAtPriceOne int64  = -1
+	)
+	poolID := node.CreateConcentratedPool(initialization.ValidatorWalletName, denom0, denom1, tickSpacing, precisionFactorAtPriceOne)
+	fmt.Println("Concentrated Pool ID: ", poolID)
+
+	concentratedPool, err := node.QueryConcentratedPool(poolID)
+	s.Require().NoError(err)
+
+	// assert contents of the pool are valid
+	s.Require().True(
```

Comment:
Can we use `s.Require.Equals for each field please?

This helps to identify what exactly is faulty if mismatch occurs